chipsec/chipset.py

```diff
@@ -933,6 +935,18 @@ def register_has_field( self, reg_name, field_name ):
             return False
         return (field_name in reg_def['FIELDS'])
 
+    def register_has_all_fields(self, reg_name, field_list):
+        try:
+            self.get_register_def(reg_name)
+        except KeyError:
+            return False
```

Comment:
it looks like lines 939-942 are repeated in register_has_field function.  Not sure if they are necessary.

Reply:
I think you are right. The way I was first wrote this, it would have returned True if the first iteration failed. I updated my loop and didn't think to take this out.